test/flatbuffer/create_bash_script.py

```diff
@@ -0,0 +1,122 @@
+import os
+import re
+
+with open('../RunTests', 'r') as file:
```

Comment:
This should be relative to the script dir and not cwd

Reply:
The files under `test/flatbuffer/` are not currently used anywhere. I think they should be checked in so that they are not lost, but created a issue to track them (https://github.com/VowpalWabbit/vowpal_wabbit/issues/2576) and use them in the CI or delete them if they can't be used.